winlogbeat/winlogbeat.full.yml

```diff
@@ -25,8 +25,8 @@
 # dictionaries.
 #
 # The supported keys are name (required), tags, fields, fields_under_root,
-# ignore_older, level, event_id, provider, and include_xml. Please visit the
```

Comment:
@andrewkroh We should probably add the full config details to the winlogbeat.full.yml config file.